README.md

```diff
@@ -20,3 +20,24 @@ odh-data-processing
         |- data-pre-processing
         |- data-post-processing
 ```
+
+## Quality & CI
+
+This repo enforces Python style and clean notebooks via **pre-commit** and a GitHub Actions workflow.
```

Comment:
You might want to remove the Github actions mention here and just mention that pre-commit can be setup with this repo.